javascript/big-practice/src/scripts/utils/selectAllDOMByClass.js

```diff
@@ -0,0 +1,3 @@
+export default function selectAllDOMClass(classes) {
```

Comment:
Should create a file to handle all cases access DOM instead of split to multiple files